src/Chapter2/Even.hs

```diff
@@ -0,0 +1,13 @@
+module Chapter2.Even where
+import Data.List
+check :: Int -> Bool
+check num = if (num `mod` 2) == 0
```

Comment:
reduce this to a one-liner. Return a `Bool` from an `if-then-else` doesn't make sense.